tests/queryset_pickle/tests.py

```diff
@@ -52,6 +52,15 @@ def test_doesnotexist_exception(self):
         self.assertEqual(original.__class__, unpickled.__class__)
         self.assertEqual(original.args, unpickled.args)
 
+    def test_can_pickle_doesnotexist_class(self):
+        """
+        #28575 - Verifies the dynamically created DoesNotExist attribute can be pickled
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Verfies that" since all tests verify things. They should also be wrapped at 79 characters and include a period.

Ticket references should be reserved for obscure issues (not needed here).

These guidelines are from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).